src/components/tables/TableItems.tsx

```diff
@@ -53,11 +53,7 @@ export const LongText = ({ tableItem, row }: TableDataElement) => {
 
 export const ShortText = ({ tableItem, row }: TableDataElement) => {
   const text = defaultUndefined(tableItem.modifier, row);
-  return (
-    <Td>
-      <Text>{text}</Text>
-    </Td>
-  );
+  return <Td>{typeof text === "string" ? <Text>{text}</Text> : text}</Td>;
```

Comment:
Why are we doing this type check?

Reply:
The type check is because the text can be a react element or a string